ui-charts/src/manchette/utils/__tests__/helpers.spec.ts

```diff
@@ -18,98 +18,58 @@ const mockedWaypoints = [
   { position: 200_000_000, id: 'waypoint-3' },
 ];
 
-describe('computeWaypointsToDisplay', () => {
-  const minZoomMillimeterPerPx = 500_000;
-  const maxZoomMillimeterPerPx = 1_000;
+describe('selectWaypointsToDisplay', () => {
   it('should ensure that a empty array is returned when there is only 1 waypoint', () => {
-    const result = computeWaypointsToDisplay(
-      [mockedWaypoints[0]],
-      {
-        height: 500,
-        isProportional: true,
-        yZoom: 1,
-      },
-      minZoomMillimeterPerPx,
-      maxZoomMillimeterPerPx
-    );
+    const result = selectWaypointsToDisplay([mockedWaypoints[0]], {
+      height: 500,
+      isProportional: true,
+      yZoom: 1,
+    });
     expect(result.length).toBe(0);
   });
 
   it('should display all points for non-proportional display', () => {
-    const result = computeWaypointsToDisplay(
-      mockedWaypoints,
-      {
-        height: 100,
-        isProportional: false,
-        yZoom: 1,
-      },
-      minZoomMillimeterPerPx,
-      maxZoomMillimeterPerPx
-    );
+    const result = selectWaypointsToDisplay(mockedWaypoints, {
+      height: 100,
+      isProportional: false,
+      yZoom: 1,
+    });
     expect(result).toHaveLength(mockedWaypoints.length);
-    expect(result[0].styles?.height).toBe(`${BASE_WAYPOINT_HEIGHT}px`);
-    expect(result[1].styles?.height).toBe(`${BASE_WAYPOINT_HEIGHT}px`);
```

Comment:
Would be nice not to lose these tests. from what I see height computation was moved to the manchette + spacetime hook, but it’s no longer covered by unit tests.

Reply:
The height is no more relevant, as the elements are now positioned absolutely. Also, their new `top` positions are computed with the GET helpers, that are derived from `spaceScalesToBinaryTree` and `getNormalizedScaleAtPosition` that have enough unit tests, in my opinion at least.